docs-chef-io/content/inspec/resources/oracle_db_conf.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "oracle_db_conf resource"
```

Comment:
For consistency with `oracledb_session`, let's rename this to be `oracledb_conf` (no additional underscore)